include/af/array.h

```diff
@@ -572,7 +572,7 @@ namespace af
         af_array get() const;
 
         /**
-           get the number of elements in array
+           Get the number of elements in array. This is the product of the array's dimensions.
```

Comment:
perhaps "Get the total number of elements across all dimensions of the array" is more succinct ?

And move the next statement as a note `\note This is the product of the array's dimensions` - Although I think this is not required given that is very basic thing with multidimensional arrays, especially if we rephrase it to the above.

Reply:
Fixed.